src/routes/login/+page.svelte

```diff
@@ -4,16 +4,22 @@
   import { goto, invalidateAll } from '$app/navigation';
   import { base } from '$app/paths';
   import { onMount } from 'svelte';
+  import { get } from 'svelte/store';
   import AlertError from '../../components/ui/AlertError.svelte';
-  import { user as userStore } from '../../stores/app';
+  import { permissibleQueries as permissibleQueriesStore, user as userStore } from '../../stores/app';
   import type { LoginResponseBody } from '../../types/auth';
 
-  let error = null;
+  let error: string | null = null;
   let loginButtonText = 'Login';
   let password = '';
   let username = '';
   let usernameInput: HTMLInputElement | null = null;
 
+  $: if (get(userStore) !== null && $permissibleQueriesStore && !Object.keys($permissibleQueriesStore).length) {
+    error =
+      'You are not authorized to access the page that you attempted to view. Please contact __someone__ to request access.';
```

Comment:
Any reason for the `__someone__`? Also noticed the error tooltip styling is pretty rough. Not something introduced by the PR but something we should fix
<img width="804" alt="image" src="https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/20bbed03-d72f-4411-b2ae-aa98bcbf2ff2">

Reply:
oh yeah lol I wasn't sure what to put there and forgot to flesh it out.

Reply:
Hm yeah.. could be something like "a tool administrator"?